Data/ByteString/Lazy.hs

```diff
@@ -1381,3 +1385,19 @@ findIndexOrEnd k (S.PS x s l) =
                                   then return n
                                   else go (ptr `plusPtr` 1) (n+1)
 {-# INLINE findIndexOrEnd #-}
+
+-- $IOChunk
+--
+-- ⚠ Using lazy I\/O functions like 'readFile' or 'hGetContents'
+-- means that the order of operations is left at the discretion
```

Comment:
Which operations?

Reply:
I meant "processing the data" and "closing the file handle" by that, but you are right, we will benefit from a clearer language here.